### PR TITLE
build: version bump to 1.1.1-0.11.3-rc1

### PR DESCRIPTION
**Pull request by loloicci**
# Description
This PR is for version bump to v1.1.1-0.11.3-rc1

## Types of changes

- [ ] Bug fix (changes which fixes an issue)
- [ ] New feature (changes which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmvm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly. (not needed)
- [ ] I have added tests to cover my changes. (not needed)
- [x] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

File: internal/api/wasmvm.dll
Comment (zemyblue) [inline internal/api/wasmvm.dll:1]: This window library did not supported, and cosmwasm/wasmvm of main branch also does not support it. 
How do you think about it?

But cosmwasm/wasmvm added it in `[window_static_lib](https://github.com/CosmWasm/wasmvm/tree/windows_static_lib/internal/api)` branch.

Comment (loloicci) [reply internal/api/wasmvm.dll:1]: I think deleting dll in our main is OK. Are there some problems if we have dll file?

Comment (zemyblue) [reply internal/api/wasmvm.dll:1]: No, I just afraided because we didn't check the wasm work well in windows.

Comment (zemyblue) [reply internal/api/wasmvm.dll:1]: My opinion is I think it's nice to delete it.

Comment (loloicci) [reply internal/api/wasmvm.dll:1]: OK, then I will delete it.

**Review by zemyblue — APPROVED** [review]

**Review by jaeseung-bae — APPROVED** [review]

**Pull request merged**